lib/std/zig/system/linux.zig

```diff
@@ -374,6 +374,13 @@ fn CpuinfoParser(comptime impl: anytype) type {
     };
 }
 
+inline fn getCpuFeature(comptime feat_reg: []const u8) u64 {
```

Comment:
Despite this being a private function the name should probably reflect that it is ARM-specific

Reply:
Done in the new version.